modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSetResultsSetTotalMethodCheck.java

```diff
@@ -43,6 +43,31 @@ protected String[] getValidExtensions() {
 		return new String[] {"java", "jsp", "jspf"};
 	}
 
+	private String _addTryCatch(
+		String newContent, String lastMethodCall, String stringBuilder) {
+
+		String newClass = StringUtil.replace(
+			newContent, lastMethodCall, stringBuilder);
+
+		Matcher matcher = _setResultsAndTotalPattern.matcher(newClass);
+
+		String setResultsAndTotalMethodCall = null;
+		StringBuilder sb = new StringBuilder();
+
+		if (matcher.find()) {
+			setResultsAndTotalMethodCall = matcher.group();
+
+			sb.append("\t\ttry {\n\t");
+			sb.append(setResultsAndTotalMethodCall);
+			sb.append("\n\t\t} catch (Throwable e) {\n");
```

Comment:
Maybe we could create a variable to the basic indentation using `JavaSourceUtil.getIndent`, and from there edit this logic.